src/lxc/conf.c

```diff
@@ -2744,8 +2744,8 @@ static int instantiate_vlan(struct lxc_handler *handler, struct lxc_netdev *netd
 	      netdev->ifindex);
 	if (netdev->mtu) {
 		int mtu = atoi(netdev->mtu);
-		err = mtu > 64 && mtu < 65536 ? lxc_netdev_set_mtu(peer,mtu) : EINVAL;
-		if(err) {
+		err = lxc_netdev_set_mtu(peer,mtu);
+		if (err) {
 			ERROR("failed to set mtu '%s' for %s : %s",
 			      netdev->mtu, peer, strerror(-err));
 			lxc_netdev_delete_by_name(peer);
```

Comment:
I'm not sure whether `lxc_netdev_set_mtu()` will return an `errno` or simply `-1`. Just remove the `strerror(-err)` and the corresponding format specifier here. Should it return an `errno` please use `strerror(err)` and not `strerror(-err)`. Thanks! :)